net_ip_socket.go

```diff
@@ -129,8 +129,7 @@ func parseIP(hexIP string) (net.IP, error) {
 	var byteIP []byte
 	byteIP, err := hex.DecodeString(hexIP)
 	if err != nil {
-		return nil, fmt.Errorf(
-			"cannot parse address field in socket line: %s", hexIP)
+		return nil, fmt.Errorf("cannot parse address field in socket line: %s", hexIP)
```

Comment:
This is presumably a hexadecimal number, but I guess it could be anything if the decoding has gone wrong. So let's use `%q` here, too?

Reply:
Yes, it's going to be a malformed string, so quoting is a good idea.